marketplace/commercelayer-sku/README.md

```diff
@@ -0,0 +1,27 @@
+# Contentful UI Extension
```

Comment:
@Shy Could you also add a `MARKETPLACE.md` file to the extension? Check out the Bynder extension as an example.

Reply:
Done.